frontend/public/components/radio.jsx

```diff
@@ -3,20 +3,29 @@ import * as React from 'react';
 import * as PropTypes from 'prop-types';
 
 export const RadioInput = (props) => {
-  const inputProps = _.omit(props, ['title', 'subTitle', 'desc', 'children']);
-  return <div className="radio-item form-group">
-    <label>
+  const inputProps = _.omit(props, ['title', 'subTitle', 'desc', 'children', 'inline']);
+  const inlineClass = props.inline ? 'radio-inline' : '';
```

Comment:
I'd just call this `labelClass` since we're always using the variable whether inline or not.

```suggestion
  const labelClass = props.inline ? 'radio-inline' : '';
```